### PR TITLE
Spot: Validation data

### PR DESCRIPTION
**Pull request by peci1**
We've finally got our hands on a real Spot, so here are the validation data.

We computed the battery endurance from the estimated percentage published by the driver which might be better suitable for linear extrapolation.

### CODE REVIEW

**Comment by peci1** [comment]
Today we measured the total mass of our sensor config. It came out about 3 kg heavier than what was estimated in the model, so I also updated the total mass of the model (this discrepancy is mostly because we found out that the internal Spot battery is not good enough to power both the robot and the sensor pack for an hour, so we decided to add a separate battery for the sensor pack).

File: submitted_models/ctu_cras_norlab_spot_sensor_config_1/model.sdf [hunk 1]
```diff
@@ -190,7 +193,7 @@
       <light name="light_left" type="spot">
         <pose>0.275 0.1 0.13 1.5707963267948966 -1.5707963267948966 0</pose>
         <cast_shadows>1</cast_shadows>
-        <attenuation>
+        <attenuation>ign sdf -p
```
Comment (acschang) [inline submitted_models/ctu_cras_norlab_spot_sensor_config_1/model.sdf:196]: Is this line intended?

**Review by acschang — CHANGES_REQUESTED** [review]

**Review by acschang — APPROVED** [review]

File: submitted_models/ctu_cras_norlab_spot_sensor_config_1/model.sdf (hunk 1 above)
Comment (acschang) [reply submitted_models/ctu_cras_norlab_spot_sensor_config_1/model.sdf:196]: Fixed in https://github.com/osrf/subt/pull/953/commits/dd1bc50e8fed275918072026f6179aa274882a50

**Comment by acschang** [comment]
Continued in #953

**Pull request closed without merging**